From f5a2b8b78afb253630ee8833f16f131b3b52701e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pawe=C5=82=20Chmielowski?= <pchmielowski@process-one.net>
Date: Mon, 29 Feb 2016 10:56:32 +0100
Subject: [PATCH] Stdint is required on windows

[Backport from upstream commit
4fc5cae81122540fff983e40dda1fa905c329fd0, which happens to not only
fix the build on Windows, but also with the musl C library.]
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 c_src/stringprep.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/c_src/stringprep.cpp b/c_src/stringprep.cpp
index 461cf09..26215cf 100644
--- a/c_src/stringprep.cpp
+++ b/c_src/stringprep.cpp
@@ -19,6 +19,7 @@
  */
 
 #include <string.h>
+#include <stdint.h>
 #include <erl_nif.h>
 
 #include "uni_data.c"
-- 
2.7.4

